pkgs/development/libraries/gstreamer/gstreamermm/default.nix

```diff
@@ -10,9 +20,22 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  nativeBuildInputs = [ pkg-config file ];
-
-  propagatedBuildInputs = [ glibmm gst_all_1.gst-plugins-base ];
+  nativeBuildInputs = [
+    pkg-config
+    file
+  ];
+
+  propagatedBuildInputs = [
+    glibmm
+    gst_all_1.gst-plugins-base
+  ];
+
+  patches = [
```

Comment:
Would be nice to keep it next to `src` as is [conventional](https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887), otherwise it is easy to miss.

Reply:
Done.